.github/workflows/autocorrect-lint.yml

```
@@ -0,0 +1,35 @@
+name: AutoCorrect Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "files/zh-cn/**"
```

I think we'd better add itself, and looks good to me now.

```suggestion
      - "files/zh-cn/**"
      - ".github/workflows/autocorrect-lint.yml"
```